packages/manager/modules/pci/src/projects/project/kubernetes/components/anti-affinity/anti-affinity.controller.js

```diff
@@ -0,0 +1,12 @@
+import has from 'lodash/has';
+import omit from 'lodash/omit';
+
+export default class NodePoolAntiAffinityCtrl {
+  onAntiAffinityChange(isAntiAffinity) {
+    if (isAntiAffinity) {
+      this.nodePool.maxNodes = 5;
+    } else if (has(this.nodePool, 'maxNodes')) {
+      omit(this.nodePool, ['maxNodes']);
```

Comment:
No need to set a new value ? Im' not sure `omit` mutates the object (https://lodash.com/docs/4.17.15#omit)